lib/cors-anywhere.js

```diff
@@ -50,8 +50,11 @@ function isValidHostName(hostname) {
  * @param headers {object} Response headers
  * @param request {ServerRequest}
  */
-function withCORS(headers, request) {
+function withCORS(headers, request, maxAge) {
   headers['access-control-allow-origin'] = '*';
```

Comment:
Sure. To do this, I had to move the definition of `req.corsAnywhereRequestState` to before the call to `withCORS` in the `getHandler` function.